packages/xo-web/src/common/xo/index.js

```diff
@@ -946,15 +946,21 @@ export const suspendVm = vm => _call('vm.suspend', { id: resolveId(vm) })
 
 export const suspendVms = vms =>
   confirm({
-    title: _('suspendVmsModalTitle', { nVms: vms.length }),
-    body: _('suspendVmsModalMessage', { nVms: vms.length }),
+    title: _('suspendVmsModalTitle', { vms: vms.length }),
+    body: _('suspendVmsModalMessage', { vms: vms.length }),
```

Comment:
IMHO it's better to use the `n` prefix to denote a count.

Reply:
Currently, we use `vms` for all the VM actions and I didn't want to add too many unrelated changes. I can remove this change and fix them all in another PR, though.

Reply:
I agree, let's take care of this later.